frontend/packages/policy-editor/src/PolicyEditor.tsx

```diff
@@ -96,15 +90,23 @@ export const PolicyEditor = ({
         <div className={classes.alertWrapper}>
           <PolicyEditorAlert />
         </div>
-        <PolicyCardRules showErrorsOnAllRulesAboveNew={showErrorsOnAllRulesAboveNew} />
-        <div className={classes.addCardButtonWrapper}>
-          <AddPolicyRuleButton onClick={handleClickAddButton} />
-        </div>
+        <PolicyEditorContent usageType={usageType} />
       </div>
     </PolicyEditorContextProvider>
   );
 };
 
+type PolicyEditorUsageContentProps = {
```

Comment:
Maybe we should keep same name as the component? 😄 
```suggestion
type PolicyEditorContentProps = {
```